pkg/types/vsphere/platform.go

```diff
@@ -5,6 +5,9 @@ type Platform struct {
 	// VCenter is the domain name or IP address of the vCenter.
 	VCenter string `json:"vCenter"`
 
+	// Port is the transportation port for vcenter server
```

Comment:
nit: comments should be complete sentences and end with `.` or proper punctuation.

This is an optional field (`omitempty`) and should have the `\\ +optional` tag, similar to the `APIVIP` field. There are some other optional fields here as well, and it would be great if you could update those with the tag. We recently added support to [generate docs based on these comments](https://github.com/openshift/installer/blob/master/docs/dev/explain.md#generating-the-documentation). 

Any changes to these types should regenerate the docs, and it would be great to update the older fields so that the docs are properly generated.